sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -263,47 +268,47 @@ const generateSingleResult = (resultData, projectName, id) => {
         page_title = highlights.title[0];
     }
 
-    // if result is not from the same project,
-    // then it must be from subproject.
+    let h2_element = createDomNode("h2", {class: "search__result__title"});
+    h2_element.innerHTML = page_title;
+
+    // If the result is not from the same project,
+    // then it's from a subproject.
     if (projectName !== resultData.project) {
-        page_title +=
-            " " +
-            render_template(
-                '<small class="rtd_ui_search_subtitle"> \
-                    (from project <%= project %>) \
-                </small>',
-                {
-                    project: resultData.project
-                }
-            );
+        let subtitle = createDomNode("small", {class: "rtd_ui_search_subtitle"});
+        subtitle.innerText = `(from project ${resultData.project})`;
+        h2_element.appendChild(subtitle);
     }
+    h2_element.appendChild(createDomNode("br"))
 
-    page_title += "<br>";
+    let a_element = createDomNode("a", {href: page_link});
+    a_element.appendChild(h2_element);
 
-    content.innerHTML += render_template(page_link_template, {
-        page_link: page_link,
-        page_title: page_title
-    });
+    let content = createDomNode("div");
+    content.appendChild(a_element);
 
+    let separator = createDomNode("br", {class: "br-for-hits"});
```

Comment:
If more space is needed, wrapping in a `<p>` is preferable to styling spacing using `<br>`, which should almost always be avoided. The issue with `<br>` is that it is difficult to style around. Wrapping each block in `<p>` should get you similar vertical spacing however.